src/node/rpc/handler_registry.h

```diff
@@ -56,12 +75,30 @@ namespace ccf
 
       nlohmann::json result_schema = nullptr;
 
+      /** Sets the JSON schema that the request response must comply with.
+       *
+       * @param j Request response JSON schema
+       * @return The installed Handler for further modification
+       */
       Handler& set_result_schema(const nlohmann::json& j)
       {
         result_schema = j;
         return *this;
       }
 
+      /** Sets the schema that the request parameters and response must comply
+       * with based on JSON-serialisable data structures.
+       *
+       * \verbatim embed:rst:leading-asterisk
+       * .. note::
+       *  See ``DECLARE_JSON_`` serialisation macros for serialising
+       *  user-defined data structures.
+       * \endverbatim
```

Comment:
I think this is cool as it can make notes/warnings really stand out, e.g.:
![image](https://user-images.githubusercontent.com/42961061/81682678-e80fdb00-944c-11ea-910e-f3f086c654f1.png)

Reply:
Nit: Is it possible to change the formatting of this function title/type? It bothers me that the spacing is different in `Handler &foo()` (vs `Handler& foo()` in the actual code).

Reply:
As discussed, this does not seem possible with Doxygen/Breathe.